internal/status.go

```diff
@@ -136,7 +136,8 @@ func (i *Internal) MonitorVICEEvents() {
 
 			deploymentInformer := factory.Apps().V1().Deployments().Informer()
 			deploymentInformerStop := make(chan struct{})
-			defer close(deploymentInformerStop)
+			// no-op, defer doesn't work in infinite loop
+			// defer close(deploymentInformerStop)
```

Comment:
Not sure if this was supposed to do something, commented out for the reason noted.